src/index.js

```diff
@@ -10,6 +10,15 @@ export default function (Vue, {
   Vue.mixin({
     beforeCreate() {
       this.$persist = (names, storeName = defaultStoreName, storeExpiration = defaultExpiration) => {
+
+        let prefix = this.$options.name;
+        if (!prefix) {
+          console.error('You must set name to Vue components while using vue-persist');
+          return;
+        }
+
+        storeName = `${prefix}__${storeName}`
```

Comment:
why do we need namespace?

Reply:
If you use vue-persist in two components with same parameters.
The second component persist data will overwrite the first one.
So we need namespace.

Reply:
In that case you should use a different store name https://github.com/egoist/vue-persist#name-1